assets/src/eejs/index.js

```diff
@@ -40,4 +40,13 @@ export { locale as SERVER_LOCALE } from './locale';
  */
 export * from './exceptions';
 
+/**
+ * Utils
+ */
 export * from './utils/merge-and-de-duplicate';
+export * from './utils/map-reducer';
+export * from './utils/convert-to-object-from-map';
+export * from './utils/is-generator';
+export * from './utils/is-factory';
+export * from './utils/is-model-entity';
+export * from './utils/is-schema';
```

Comment:
exposed a bunch new utility functions on the `eejs` global (within src can be imported from `@eventespresso/eejs`)